java/lang/String/String.cpp

```diff
@@ -697,6 +697,19 @@ String String::operator+(const string &target) {
 }
 
 /**
+ * Operator plus for const string
+ *
+ * @param target
+ * @return

```

Comment:
I follow this documentation, [How to write doc comment for Java Doc Tool](http://www.oracle.com/technetwork/articles/java/index-137868.html)
Please show me the rule Data type after `@String` and I will fix it